apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-title/index.js

```diff
@@ -0,0 +1,27 @@
+/* global fullSiteEditing */
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
```

Comment:
I think this will need some styles too, imported here: `import './style.scss';`.